src/main/java/uk/nhs/digital/nhsconnect/nhais/model/mesh/MeshMessage.java

```diff
@@ -8,15 +8,39 @@
 @Getter @Setter @ToString @EqualsAndHashCode
 public class MeshMessage {
 
-    private String odsCode;
+    /**
+     * If SENDING TO MESH: set to the HA Trading Partner Code (recipient)
+     * If DOWNLOADING FROM MESH: DO NOT USE
```

Comment:
So maybe we should have `InboundMeshMessage` and `OutboundMeshMessage`?

Reply:
Probably we should, maybe we could use interfaces for that, but I don't think it's that important right now